client/src/stores/game.ts

```diff
@@ -86,13 +86,13 @@ export const useGameStore = defineStore('game', {
   getters: {
     isSelector(): boolean {
       const room = useRoomStore();
-      return this.selector?.id === room.self!.id;
+      return this.selector?.id === room.self.id;
```

Comment:
it seems most of your errors come from a mass refactor removal of `!`, looks like you need to add back the `!`